bench-templates/src/macros/field.rs

```diff
@@ -1,542 +1,468 @@
+            let v_bits_be = v1
+                .iter()
+                .map(|s| ark_ff::BitIteratorBE::new(s).collect::<Vec<_>>())
+                .collect::<Vec<_>>();
+            bits.bench_function(
+                &format!("From Big-Endian bits for {}::BigInt", stringify!($F)),
+                |b| {
+                    let mut i = 0;
+                    b.iter(|| {
+                        i = (i + 1) % SAMPLES;
+                        BigInt::from_bits_be(&v_bits_be[i]);
+                    })
+                },
+            );
+            bits.bench_function(&format!("Comparison for {}::BigInt", stringify!($F)), |b| {
+                let mut i = 0;
+                b.iter(|| {
+                    i = (i + 1) % SAMPLES;
+                    v1[i] > v2[i]
                 })
-                .collect();
-
-            let mut count = 0;
-            b.iter(|| {
-                let mut tmp = v[count].0;
-                n_fold!(tmp, v, sub_with_borrow, count);
-                count = (count + 1) % SAMPLES;
-                tmp;
-            });
-        }
-
-        fn repr_num_bits(b: &mut $crate::bencher::Bencher) {
-            const SAMPLES: usize = 1000;
-
-            let mut rng = ark_std::test_rng();
-
-            let v: Vec<$f_repr_type> = (0..SAMPLES).map(|_| $f_repr::rand(&mut rng)).collect();
-
-            let mut count = 0;
-            b.iter(|| {
-                let tmp = v[count].num_bits();
-                count = (count + 1) % SAMPLES;
-                tmp;
             });
-        }
-
-        fn repr_mul2(b: &mut $crate::bencher::Bencher) {
-            const SAMPLES: usize = 1000;
-
-            let mut rng = ark_std::test_rng();
-
-            let v: Vec<$f_repr_type> = (0..SAMPLES).map(|_| $f_repr::rand(&mut rng)).collect();
-
-            let mut count = 0;
-            b.iter(|| {
-                let mut tmp = v[count];
-                n_fold!(tmp, mul2);
-                count = (count + 1) % SAMPLES;
-                tmp;
-            });
-        }
-
-        fn repr_div2(b: &mut $crate::bencher::Bencher) {
-            const SAMPLES: usize = 1000;
-
-            let mut rng = ark_std::test_rng();
-
-            let v: Vec<$f_repr_type> = (0..SAMPLES).map(|_| $f_repr::rand(&mut rng)).collect();
-
-            let mut count = 0;
-            b.iter(|| {
-                let mut tmp = v[count];
-                n_fold!(tmp, div2);
-                count = (count + 1) % SAMPLES;
-                tmp;
+            bits.bench_function(&format!("Equality for {}::BigInt", stringify!($F)), |b| {
+                let mut i = 0;
+                b.iter(|| {
+                    i = (i + 1) % SAMPLES;
+                    v1[i] == v2[i]
+                })
             });
         }
 
-        fn into_repr(b: &mut $crate::bencher::Bencher) {
-            const SAMPLES: usize = 1000;
-
-            let mut rng = ark_std::test_rng();
+        // fn into_repr(b: &mut $crate::bencher::Bencher) {
```

Comment:
Plan?

Reply:
Fixed in [`b0e862b` (#472)](https://github.com/arkworks-rs/algebra/pull/472/commits/b0e862b405e3114ea353432a4c273defbb7e7c88)